pup-0005.md

```diff
@@ -50,16 +51,17 @@ opposed to an asymmetrical contributor license agreement); and (3)
 traditionally Red Hat developers have exercised broad discretion on
 project licensing policy.
 
-Pulp is a good candidate for adoption of the CCRC at the project
+Pulp is a good candidate for adoption of the GPLCC at the project
 level. Pulp is a GPLv2-licensed community project of commercial
 significance to Red Hat, and Pulp has been making significant efforts
 to build a community around the project, which over time will
 encourage contributions to the project from various copyright holders.
 
-## Common Cure Rights Commitment, version 1.0
+## GPL Cooperation Commitment, Version 1.0
```

Comment:
Do we need to bump the version number to 2 here on line 60 and below on line 63 as well to match line 6?

Reply:
No, I think one is the version # of the PUP and this is the version number of GPLCC.

Reply:
Ah, gotcha, ok, then nevermind, it's good as is.